apis/cloud.redhat.com/v1alpha1/clowdapp_types.go

```diff
@@ -68,7 +68,13 @@ type Job struct {
 	Name string `json:"name"`
 
 	// Defines the schedule for the job to run
-	Schedule string `json:"schedule"`
+	Schedule string `json:"schedule,omitempty"`
+
+	// A job that is a OneShot is run on apply
+	OneShot bool `json:"oneshot,omitempty"`
```

Comment:
>  if we are never going to have both oneshot and onrequest set together,
Hmm. I'm not sure this will be the case. Are there instances where a job that runs on deploy would want to be run again in the future?